README.md

```diff
@@ -61,7 +62,7 @@ Additional features can be enabled on demand.
   Cross Account VPC Peering support,
   IPv6 support,
   VPC Flow Logs,
-  Subnet group support for AWS services like RDS and Redis,
+  Subnet group support for Redis,
```

Comment:
Did we really only add subnet group support for Redis?

Reply:
no, we only added it for RDS now... elasticache is missing..as all the others are also still missing..

Reply:
so then it should say RDS, right? 

>   Subnet group support for Redis,

Reply:
that's missing features

Reply:
ah true :)